hugo.yaml

```diff
@@ -21,6 +22,7 @@ languages:
       description: OpenTelemetryプロジェクト公式サイト
   zh:
     languageName: 中文 (Chinese)
+    languageCode: zh-cn
```

Comment:
@open-telemetry/docs-zh-approvers - is this ok with you?

Reply:
lgtm